### PR TITLE
fix(react-tags-preview): use regular icon for dismiss

### PR DESCRIPTION
**Pull request by YuanboXue-Amber**
Use regular icon for dismiss icon instead of bundled icon, since we don't need the filled icon anyway. 

This PR also fixes several issue in stories discovered in a11y test:
1. add aria-label for disabled Tag/InteractionTag story
2. add aria-label on TagGroup for TagGroup stories.
3. use just "dismiss"/"remove" for InteractionTagSecondary aria-label

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5733e035062b24e0209674c848b8a6f058390263'>5733e035062b24e0209674c848b8a6f058390263</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-tags-preview/stories/TagGroup/TagGroupSizes.stories.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const Sizes = () => {
       {sizes.map(size => (
         <div key={size}>
           {`${size}: `}
-          <TagGroup size={size}>
+          <TagGroup size={size} aria-label="Sizes">
```
Comment (ling1726) [inline packages/react-components/react-tags-preview/stories/TagGroup/TagGroupSizes.stories.tsx:29]: is this really an accessible label? what should I interpret from this control with `Sizes`? 😅

File: packages/react-components/react-tags-preview/stories/TagGroup/TagGroupDefault.stories.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { TagGroup, Tag, TagGroupProps } from '@fluentui/react-tags-preview';
 
 export const Default = (props: Partial<TagGroupProps>) => (
-  <TagGroup {...props}>
+  <TagGroup {...props} aria-label="Default">
```
Comment (ling1726) [inline packages/react-components/react-tags-preview/stories/TagGroup/TagGroupDefault.stories.tsx:5]:
```suggestion
  <TagGroup {...props} aria-label="Simple tag group example">
```

File: packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDisabled.stories.tsx [hunk 3]
```diff
@@ -17,19 +17,19 @@ export const Disabled = () => {
...
         </InteractionTagPrimary>
-        <InteractionTagSecondary />
+        <InteractionTagSecondary aria-label="outline disabled, remove" />
       </InteractionTag>
       <InteractionTag disabled appearance="brand">
         <InteractionTagPrimary secondaryText="appearance=brand" icon={<CalendarMonthRegular />} hasSecondaryAction>
           disabled
         </InteractionTagPrimary>
-        <InteractionTagSecondary />
+        <InteractionTagSecondary aria-label="brand disabled, remove" />
```
Comment (ling1726) [inline packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDisabled.stories.tsx:32]:
```suggestion
        <InteractionTagSecondary aria-label=" remove" />
```

The differences in each of these examples is visual right? in that case we probaly don't need the appearances in label here. 

Also is `disabled` not obvious for the secondary action that we need to put a label for it? that seems kinda scary

Comment (YuanboXue-Amber) [reply packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDisabled.stories.tsx:32]: So Milan helped me did the static analysis of a11y and found out this example has no label. But we need the 2nd round - the screen reader test to know what's the correct label. I'm just putting something I think it's correct there. Let me update this to just `remove` and see if this is good enough on the screen reader tests

File: packages/react-components/react-tags-preview/stories/TagGroup/TagGroupSizes.stories.tsx (hunk 1 above)
Comment (YuanboXue-Amber) [reply packages/react-components/react-tags-preview/stories/TagGroup/TagGroupSizes.stories.tsx:29]: :D Oops, let me update it

File: packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDisabled.stories.tsx (hunk 3 above)
Comment (YuanboXue-Amber) [reply packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDisabled.stories.tsx:32]: Actually other stories has for example `<InteractionTagSecondary aria-label="filled, remove" />`. I'll update all stories to have just `<InteractionTagSecondary aria-label="remove" />`.

**Review by ling1726 — APPROVED** [review]

**Pull request merged**